Content.Server/Mech/Equipment/EntitySystems/MechGrabberSystem.cs

```diff
@@ -39,6 +40,8 @@ public override void Initialize()
         SubscribeLocalEvent<MechGrabberComponent, MechEquipmentUiStateReadyEvent>(OnUiStateReady);
         SubscribeLocalEvent<MechGrabberComponent, MechEquipmentRemovedEvent>(OnEquipmentRemoved);
         SubscribeLocalEvent<MechGrabberComponent, AttemptRemoveMechEquipmentEvent>(OnAttemptRemove);
+        SubscribeLocalEvent<MechGrabberComponent, EntInsertedIntoContainerMessage>(OnEntityInserted);
+        SubscribeLocalEvent<MechGrabberComponent, EntRemovedFromContainerMessage>(OnEntityRemoved);
 
```

Comment:
Mech equipment code is already bad but shouldn't this just be checking if the entitystorage is in a container instead as otherwise you're going to need to copy this fix every single time someone puts storage into a container.

Reply:
I feel stupid for not knowing that system function existed